### PR TITLE
DOCSP-43918 - Removed APIs

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-43918

### Staging Links

<li><a href=https://deploy-preview-243--mongodb-docs-csharp.netlify.app/upgrade/v3>upgrade/v3</a></li>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/upgrade/v3.txt [hunk 1]
```diff
@@ -61,6 +61,11 @@ Version 3.0 Breaking Changes
   and {+mdb-server+} versions, visit the
   :ref:`Compatibility <csharp-compatibility-tables>` page.
 
+- The classes, methods, and properties ``MongoDB.Driver.Core`` namespace that were
```
Comment (mcmorisi) [inline source/upgrade/v3.txt:64]: Should this be "classes, methods, and properties _in the_ ``MongoDB.Driver.Core`` namespace"?

Comment (mongoKart) [reply source/upgrade/v3.txt:64]: yep! good catch.

Comment (mongoKart) [inline source/upgrade/v3.txt:64]:
```suggestion
- The classes, methods, and properties in the ``MongoDB.Driver.Core`` namespace that were
```

**Review by mcmorisi — APPROVED** [review]
LGTM!

**Review by BorisDog — APPROVED** [review]
LGTM

**Pull request merged**